ci/setup_conda.sh

```diff
@@ -41,6 +41,9 @@ do
 
     . $CONDA_PATH/etc/profile.d/conda.sh && conda activate py${python_version}
     echo "Python version: $(python --version)"
+    pip install PyYAML
```

Comment:
You can also remove pip install instructions from there because this script is only for the base docker image (base) and it won't be updated in this PR as we keep these packages installations in `build_python.sh`  script